frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -395,6 +428,23 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
         ) : (
           <Spinner animation="border"></Spinner>
         );
+      case SidebarContextType.ADD_SUBDIVISION_LAND:
+        return (
+          <LandForm
+            setMovingPinNameSpace={setMovingPinNameSpace}
+            formikRef={formikRef}
+            handleGeocoderChanges={handleGeocoderChanges}
+            handlePidChange={handlePidChange}
+            handlePinChange={handlePinChange}
+            findMatchingPid={findMatchingPid}
+            isPropertyAdmin={keycloak.hasClaim(Claims.ADMIN_PROPERTIES)}
+            setLandComplete={setShowCompleteModal}
+            setLandUpdateComplete={setShowUpdatedModal}
+            initialValues={
+              parcelDetail ?? { ...getInitialValues(), propertyTypeId: PropertyTypes.SUBDIVISION }
+            }
+          />
+        );
       case SidebarContextType.ADD_BARE_LAND:
       case SidebarContextType.UPDATE_DEVELOPED_LAND:
       case SidebarContextType.UPDATE_BARE_LAND:
```

Comment:
reusing this context for now, since there is no data model difference between parcels and subdivisions. May need to separate this out later.